README.md

```diff
@@ -1,27 +1,32 @@
-# Project Name
+# Fcc Cat Photo App
```

Comment:
Since "Fcc" is an abbreviation, why don't we make it all caps?

Reply:
okay sir